### DISCUSSION TITLE
Graph API : Search People (beta) : returning incorrect "Total" in response

### DISCUSSION THREAD

**Discussion by harshasyamala** [microsoft-search/pnp-modern-search#2904] [Q&A]
MS Graph API : People Search endpoint is returning incorrect values in "Total" field of the response. This is causing a problem when Paging is enabled on PnP Modern Search webpart - people results use cases. 

For example, when searching for people (using graph explorer) for 

**Case 1:** keyword "test", (without specifying any specific size) the total number of people returned are 7
-------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/28626579/231419286-36865360-e602-41a7-a715-d3576ad8c3a5.png)

**Case 2:** keyword "test", (with specifying "3" size) to get 3 results per page. Issue is the "total" changed to 3 instead of actual 7. 
---------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/28626579/231419475-5596c96b-fd5e-469c-99ec-191d2ba0e122.png)

Question here is, is this an expected behaviour or am I missing something here ?

**Comment by kasperbolarsen** [comment]
The people search API is still beta AFAIK,  and I raised an issue with the Graph team as the API couldn't handle a wildcard search 🤣
I guess we will just have to wait.

**Reply by wobba** [reply]
Total counts are estimates. Either way, this is API related. Graph questions are best logged as described via https://developer.microsoft.com/en-us/graph/support
